cfme/common/__init__.py

```diff
@@ -75,13 +91,17 @@ def _assign_unassign_policy_profiles(self, assign, *policy_profile_names):
             policy_profile_names: :py:class:`str` with Policy Profile names.
         """
         view = navigate_to(self, 'ManagePoliciesFromDetails')
-        for policy_profile in policy_profile_names:
-            if assign:
-
-                view.policy_profiles.check_node(policy_profile)
-            else:
-                view.policy_profiles.uncheck_node(policy_profile)
-        view.save.click()
+        # Save button will be disabled in case of no changes done to policy,
+        # check if need at list one policy changes, else click on Cancel
+        if view.check_policy_assign_unassign(assign, *policy_profile_names):
+            for policy_profile in policy_profile_names:
+                if assign:
```

Comment:
This is implemented in two separate PR's, and I think it should really be implemented in `CheckableBootstrapTreeview` directly within its `fill()` method.

Reply:
implement by CheckableBootstrapTreeview directly within its fill() method